src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+export function rounder(longNumber: number): number {
+  return Math.floor(longNumber * 100) / 100;
+}
 
+export type Color = 'red' | 'green' | 'blue';
+
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number;
 }
 
 export class Triangle {
+  public shape = Shape.Triangle;
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if ([a, b, c].some((number) => number <= 0)) {
+      throw new Error('One of the sides less than 1');
+    }
```

Comment:
You did this check for every initial params in every class
You can create separate function that can accept any numbers of params and check if every of this param > 0